e2e-test/src/test/java/org/alfresco/hxi_connector/e2e_test/UpdateNodeE2eTest.java

```diff
@@ -67,49 +90,54 @@ public class UpdateNodeE2eTest
     @Container
     static final AlfrescoRepositoryContainer repository = createRepositoryContainer();
     @Container
-    private static final LocalStackContainer awsMock = DockerContainers.createLocalStackContainerWithin(network);
+    private static final WireMockContainer hxInsightMock = DockerContainers.createWireMockContainerWithin(network)
+            .withFileSystemBind("src/test/resources/wiremock/hxinsight", "/home/wiremock", BindMode.READ_ONLY);
     @Container
     private static final GenericContainer<?> predictionApplier = createPredictionApplierContainer();
 
     RepositoryNodesClient repositoryNodesClient = new RepositoryNodesClient(repository.getBaseUrl(), "admin", "admin");
 
     @BeforeAll
-    public static void beforeAll() throws IOException, InterruptedException
+    public static void beforeAll()
     {
-        awsMock.execInContainer("awslocal", "sqs", "create-queue", "--queue-name", QUEUE_NAME);
+        WireMock.configureFor(hxInsightMock.getHost(), hxInsightMock.getPort());
     }
 
     @Test
     @SuppressWarnings("PMD.JUnitTestsShouldIncludeAssert")
-    void testUpdateFile() throws IOException
+    void testApplyPredictionToNode() throws IOException
     {
         // given
         @Cleanup
         InputStream fileContent = new ByteArrayInputStream(DUMMY_CONTENT.getBytes());
-        Node createdNode = repositoryNodesClient.createFileNode("-my-", "dummy.txt", fileContent, "text/plaint");
+        Node createdNode = repositoryNodesClient.createNodeWithContent("-my-", "dummy.txt", fileContent, "text/plaint");
```

Comment:
I guess this doesn't do anything except get passed through, but I think it should be `text/plain`.

Reply:
Thanks for spotting that! Fixed